### ISSUE TITLE
Would you please silence unused variable 'name'?

### ISSUE THREAD

**Issue by sweihub** [stephenberry/glaze#463]
Hello,

Thanks for the excellent JSON library, I upgraded the main today, but have a unused warning, would you silence it?

```cpp
/home/swei/quant/core/src/trading_hours.cpp:51:1: warning: unused variable 'name' [-Wunused-const-variable]
...
/home/swei/quant/core/include/glaze/core/macros.hpp:34:41: note: expanded from macro 'GLZ_META'
      static constexpr std::string_view name = #C;
```

My Best

**Comment by stephenberry** [comment]
This is needed for the macro to be used with other parts of Glaze, such as JSON schema. However, I can add the attribute [[maybe_used]] so that your warnings go away. From my experience the compiler won't include these static constexpr strings if they aren't used.

**Comment by stephenberry** [comment]
Fixed in #464
